src/lib.rs

```diff
@@ -145,5 +152,6 @@ async fn get_authentication_manager(
 ///
 /// Returns `AuthenticationManager` which can be used to obtain tokens
 pub async fn init() -> Result<AuthenticationManager, Error> {
+    log::debug!("initialising gcp_auth");
```

Comment:
Nit: make this `Initializing...` to (a) make capitalization consistent with the other log messages you've added and (b) use US spelling?

Reply:
heh, I tend to prefer UK spelling but sure :)